turtlebot2_drivers/CMakeLists.txt

```diff
@@ -12,28 +12,34 @@ find_package(ament_cmake REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(rmw_implementation_cmake REQUIRED)
-
-include_directories(${kobuki_driver_INCLUDE_DIRS})
-
-macro(targets)
-  if(NOT target_suffix STREQUAL "")
-    get_rclcpp_information("${rmw_implementation}" "rclcpp${target_suffix}")
-  endif()
-  add_executable(kobuki_node${target_suffix} src/kobuki_node.cpp)
-  ament_target_dependencies(kobuki_node${target_suffix}
-    "rclcpp${target_suffix}"
-    "nav_msgs"
-    "geometry_msgs")
-  target_link_libraries(kobuki_node${target_suffix} ${kobuki_driver_LIBRARIES})
-  add_executable(dumb_teleop${target_suffix} src/dumb_teleop.cpp)
-  ament_target_dependencies(dumb_teleop${target_suffix}
-    "rclcpp${target_suffix}"
-    "nav_msgs"
-    "geometry_msgs")
-
-  install(TARGETS kobuki_node${target_suffix} dumb_teleop${target_suffix}
-          DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
-endmacro()
-
-call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+find_package(sensor_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
+
+add_executable(kobuki_node src/kobuki_node.cpp)
+ament_target_dependencies(kobuki_node
+  "geometry_msgs"
+  "nav_msgs"
+  "rclcpp"
+  "sensor_msgs"
+  "tf2_ros")
+# Note that this setting must come after the call to
+# ament_target_dependencies() to avoid including a ROS1 tf2 installation that
+# might be on the system.
+target_include_directories(kobuki_node PRIVATE ${kobuki_driver_INCLUDE_DIRS})
```

Comment:
Or, to check what we're using from `/opt/ros`:

~~~
$ ldd `which kobuki_node` | grep opt
	libkobuki.so => /opt/ros/kinetic/lib/libkobuki.so (0x00007f53eb714000)
	libecl_mobile_robot.so => /opt/ros/kinetic/lib/libecl_mobile_robot.so (0x00007f53e9715000)
	libecl_devices.so => /opt/ros/kinetic/lib/libecl_devices.so (0x00007f53e94f6000)
	libecl_geometry.so => /opt/ros/kinetic/lib/libecl_geometry.so (0x00007f53e92d2000)
	libecl_threads.so => /opt/ros/kinetic/lib/libecl_threads.so (0x00007f53e90c7000)
	libecl_time.so => /opt/ros/kinetic/lib/libecl_time.so (0x00007f53e8ebe000)
	libecl_exceptions.so => /opt/ros/kinetic/lib/libecl_exceptions.so (0x00007f53e8cb8000)
	libecl_type_traits.so => /opt/ros/kinetic/lib/libecl_type_traits.so (0x00007f53e74c6000)
	libecl_time_lite.so => /opt/ros/kinetic/lib/libecl_time_lite.so (0x00007f53e72bf000)
~~~